### PR TITLE
oc: warn about missing stream when deleting a tag

### PR DESCRIPTION
**Pull request by kargakis**
@soltysh @miminar PTAL

### CODE REVIEW

File: pkg/cmd/cli/cmd/tag.go [hunk 1]
```diff
@@ -267,7 +267,7 @@ func (o TagOptions) Validate() error {
 	}
 
 	if o.deleteTag && o.aliasTag {
-		return errors.New("--alias and --delete may not both be specified")
+		return errors.New("--alias and --delete should not be used together")
```
Comment (miminar) [inline pkg/cmd/cli/cmd/tag.go]: `should` leaves a seed of doubt that it might actually work.

Comment (soltysh) [inline pkg/cmd/cli/cmd/tag.go]: Yeah, go with the old form, the only thing you can fix is `may not be both specified`

**Comment by soltysh** [comment]
Other than this small nit LGTM.

File: pkg/cmd/cli/cmd/tag.go (hunk 1 above)
Comment (kargakis) [inline pkg/cmd/cli/cmd/tag.go]: ok, updated to `may not be both specified`

**Comment by kargakis** [comment]
comments addressed, [merge]

**Pull request merged**